go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/networkservicemesh/cmd-forwarder-sriov
+
+go 1.14
+
+require (
+	github.com/OneOfOne/xxhash v1.2.8 // indirect
```

Comment:
Please trim out the // indirect lines, they are usually an artifact of go get pollution.